src/Lib/Canvas.vala

```diff
@@ -201,10 +230,22 @@ public class Akira.Lib.Canvas : Goo.Canvas {
 
         holding = false;
 
+        if (event.button == Gdk.BUTTON_MIDDLE) {
+            edit_mode = EditMode.MODE_SELECTION;
+        }
+
         //item_moved (selected_item);
         //add_hover_effect (selected_item);
```

Comment:
Do we need these? Maybe we can remove them.

Reply:
No, just a leftover! Gonna remove it!